frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
+      cy.install();
+    cy.login();
+    cy.visit('/dashboards/object-service');
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Test admin user login for noobaa', () => {
```

Comment:
We dont want to modify existing test cases but migrate them to cypress.
https://github.com/openshift/console/blob/36558f21ee5161b36e6da6d19ef0da2adaf8cf28/frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/noobaa-sso-scenario.ts#L1